app/src/main/java/org/mozilla/fenix/compose/Chip.kt

```diff
@@ -0,0 +1,196 @@
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Default layout for a clickable chip.
+ *
+ * @param text [String] displayed in this chip.
+ * @param modifier [Modifier] used to be applied to the layout of the chip.
+ * @param textColor Optional text [Color] for the chip.
+ * @param backgroundColor Optional background [Color] for the chip.
+ * @param onClick Callback for when the user taps this chip.
+ */
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+private fun Chip(
+    text: String,
+    modifier: Modifier = Modifier.clip(RoundedCornerShape(25.dp)),
+    textColor: Color = FirefoxTheme.colors.textActionPrimary,
+    backgroundColor: Color = FirefoxTheme.colors.actionPrimary,
+    onClick: () -> Unit,
+) {
+    androidx.compose.material.Chip(
+        onClick = onClick,
+        modifier = modifier,
+        colors = chipColors(
+            backgroundColor = backgroundColor,
+            contentColor = textColor,
+        ),
+    ) {
+        Text(
+            text = text,
+            color = textColor,
+            style = FirefoxTheme.typography.body2,
+        )
+    }
+}
+
+/**
+ * Default layout of a selectable chip.
+ *
+ * @param text [String] displayed in this chip.
+ * @param isSelected Whether this should be shown as selected.
+ * @param selectableChipColors The color set defined by [SelectableChipColors] used to style the chip.
+ * @param onClick Callback for when the user taps this.
+ */
+@Composable
+fun SelectableChip(
+    text: String,
+    isSelected: Boolean,
+    selectableChipColors: SelectableChipColors = SelectableChipColors.buildColors(),
+    onClick: () -> Unit,
+) {
+    var selected by remember { mutableStateOf(isSelected) }
+
+    Chip(
+        text = text,
+        textColor = if (selected) selectableChipColors.selectedTextColor else selectableChipColors.unselectedTextColor,
+        backgroundColor = if (selected) {
+            selectableChipColors.selectedBackgroundColor
+        } else {
+            selectableChipColors.unselectedBackgroundColor
+        },
+    ) {
+        selected = !selected
+        onClick()
+    }
+}
+
+/**
+ * Wrapper for the color parameters of [SelectableChip].
+ *
+ * @param selectedTextColor Text [Color] when the chip is selected.
+ * @param unselectedTextColor Text [Color] when the chip is not selected.
+ * @param selectedBackgroundColor Background [Color] when the chip is selected.
+ * @param unselectedBackgroundColor Background [Color] when the chip is not selected.
```

Comment:
```suggestion
 * @param selectedBackgroundColor Background [Color] when the chip is selected.
 * @param unselectedBackgroundColor Background [Color] when the chip is not selected.
 * @param selectedTextColor Text [Color] when the chip is selected.
 * @param unselectedTextColor Text [Color] when the chip is not selected.
```
The param ordering for text/background are swapped.